app/scripts/lib/transaction/metrics.ts

```diff
@@ -716,12 +716,11 @@ function getUniqueId(
   eventName: TransactionMetaMetricsEvent,
   transactionId: string,
 ) {
-  const isSubmitted = [
-    TransactionMetaMetricsEvent.finalized,
-    TransactionMetaMetricsEvent.submitted,
-  ].includes(eventName);
+  const isFinalizedOrSubmitted =
```

Comment:
The root cause of the error is that we're trying to compare an enum value (eventName) with individual enum members (finalized and submitted) directly using the includes method.